docs/removed.md

```diff
@@ -1,5 +1,46 @@
 ---
 layout: group
+title: QUnit.extend()
```

Comment:
This is the index file for the group, similar to `deprecated.md` and `extension/index.md`. It will list the pages that use that group.

I think removing `extend` from the extension group is fine, but perhaps keep it in that directory for now for its URL continuity. We could possibly move it to a new directory as well for removed things if we want the URL to change (and keep a redirect), but this file is not meant to be an entry by itself :)

Reply:
Should I create a `removed` folder and move the file there? I removed the docs change commit for now ;)